src/main/resources/translator.properties

```diff
@@ -47,6 +47,19 @@ logRotator.daysToKeep.type = METHOD
 configure = configure
 configure.type = OBJECT
 
+org.jenkinsci.plugins.buildnamesetter.BuildNameSetter = buildNameSetter
+org.jenkinsci.plugins.buildnamesetter.BuildNameSetter.type = OBJECT
+
+hudson.plugins.throttleconcurrents.ThrottleJobProperty = throttleConcurrentBuilds
+maxConcurrentPerNode = maxPerNode
+maxConcurrentTotal = maxTotal
+throttleEnabled = throttleDisabled

```

Comment:
This doesn't seem right. `ThrottleJobProperty` will require a custom strategy to handle this inverse logic.

Reply:
Sorry, managed to wander away for 6 months. I've got a PR that should work that I'll open shortly

Reply:
Breadcrumbs: https://github.com/jenkinsci/xml-job-to-job-dsl-plugin/pull/17